art/estimators/classification/keras.py

```diff
@@ -499,6 +524,28 @@ def get_activations(self, x, layer, batch_size):
 
         return activations
 
+    def custom_gradient(self, nn_function, tensors, input_values):
```

Comment:
Please make the method name more specific.
```suggestion
    def custom_loss_gradient(self, nn_function, tensors, input_values):
```